js/packages/openinference-instrumentation-llama-index/src/instrumentation.ts

```diff
@@ -51,6 +56,18 @@ export class LlamaIndexInstrumentation extends InstrumentationBase<
     return module;
   }
 
+  private isRetriever(retriever: unknown): retriever is BaseRetriever {
+    return !!retriever && !!(retriever as BaseRetriever).retrieve;
+  }
+
+  private isEmbedding(value: unknown): value is BaseEmbedding {
+    return (
+      !!value &&
+      value instanceof BaseEmbedding &&
+      !!(value as BaseEmbedding).getQueryEmbedding
+    );
+  }
+
```

Comment:
these don't need to be on the class, can move to utils and write some tests for them as well

Reply:
Will be moved on llm branch!